src/ProjectTemplates/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@microsoft/extensions-project-templates",
+  "version": "1.0.0",
+  "description": "Project template dependencies",
+  "main": "index.js",
+  "config": {
+    "destRoot": {
+      "chat": "./Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData.Web/wwwroot"
+    }
+  },
+  "scripts": {
+    "copy-dependency": "node copyDependency.mjs",
+    "copy-dependencies": "npm run copy-dompurify && npm run copy-marked && npm run copy-tailwindcss && npm run copy-pdfjs",
+    "copy-dompurify": "npm run copy-dependency -- --name dompurify --dest-root chat --src-root dist --files purify.es.mjs;purify.es.mjs.map",
+    "copy-marked": "npm run copy-dependency -- --name marked --dest-root chat --src-root lib --files marked.esm.js;marked.esm.js.map",
+    "copy-tailwindcss": "npm run copy-dependency -- --name tailwindcss --dest-root chat --files preflight.css",
+    "copy-pdfjs": "npm run copy-dependency -- --name pdfjs-dist --dest-root chat --files build/pdf.mjs;build/pdf.mjs.map;build/pdf.worker.mjs;build/pdf.worker.mjs.map;web/pdf_viewer.css;web/pdf_viewer.mjs;web/pdf_viewer.mjs.map;web/images/loading-icon.gif"
```

Comment:
How would you feel about omitting all the `.map` files? I don't think that we need to do anything for debugging these files since they are effectively vendor binaries. Would be nice just to minimize the number of files people have to make sense of.

Vaguely related, what about putting these vendor-type dependencies into `wwwroot/lib/*` instead of `wwwroot/*`? Just to make it easier to distinguish them from the developer's own folder structure of custom wwwroot content.

Reply:
> How would you feel about omitting all the .map files?

I was thinking about doing this as well. If we do that, we could probably also opt to use minified JS when available.

> Vaguely related, what about putting these vendor-type dependencies into wwwroot/lib/* instead of wwwroot/*?

Agreed. I'll change that too.